### ISSUE TITLE
Mastodon: Alt-text field needs a character counter

### ISSUE THREAD

**Issue by Mohamed00** [manuelcortez/TWBlue#538]
Hi, currently, the dialog that ask you to provide a description for media upload is lacking a character counter. It would be nice if this was added, because there's often a character limit on alt text, usually about 1500.

**Comment by manuelcortez** [comment]
Hi, do you have documentation regarding this limit? I have been reading official Mastodon docs and did not find anything regarding this.

**Comment by Mohamed00** [comment]
Not exactly, but I did try to describe something at one point, and the instance I was on returned an error saying that the description was too long and had a maximum limit of 1500 characters.

**Comment by Mohamed00** [comment]
Though perhaps if you can't find any documentation on the alt text limit, it might be useful to add an edit button for the description when viewing attachments. Perhaps that field should also be multiline?
